package.json

```diff
@@ -57,8 +58,11 @@
     "eslint-plugin-import": "^2.17.3",
     "mocha": "^5.2.0",
     "nodemon": "^1.19.1",
-    "nyc": "^14.1.1",
-    "sinon": "^7.3.2"
+    "nyc": "^14.1.1"
+  },
+  "engines": {
+    "node": ">=6.0.0",
+    "npm": ">1.3.13"
```

Comment:
why version `1.3` ? version `6` is deff stable

Reply:
I wanted it to be backward compatible. Should I lock it to version 6 and above?